tools/user_migration/migrate_node.py

```diff
@@ -558,26 +571,27 @@ def get_args():
     parser.add_argument(
         '--start_bso', default=0,
         type=int,
-        help="start dumping BSO database"
+        help="start dumping BSO database (default: 0)"
     )
     parser.add_argument(
         '--end_bso',
         type=int, default=19,
-        help="last BSO database to dump"
+        help="last BSO database to dump (default: 19)"
     )
     parser.add_argument(
         '--fxa_file',
         default="users.csv",
-        help="FXA User info in CSV format"
+        help="FXA User info in CSV format (default users.csv)"
     )
     parser.add_argument(
         '--skip_collections', action='store_false',
         help="skip user_collections table"
     )
     parser.add_argument(
-        '--readchunk',
+        '--write_chunk',
+        dest="chunk",
         default=1000,
```

Comment:
Sure, I think I picked 1K mostly because it didn't bump against the migration limit, but if it chokes we can have them pick a different value and change it retroactively.